src/react/databrowser/objects/ObjectListTable.jsx

```diff
@@ -73,11 +74,42 @@ export default function ObjectListTable({
   const handleCellClicked = useCallback(
     (bucketName, key) => e => {
       e.stopPropagation();
-      dispatch(push(`/buckets/${bucketName}/objects/${key}`));
+      query.set('prefix', key);
+      dispatch(push(`/buckets/${bucketName}/objects?${query.toString()}`));
     },
     [dispatch],
```

Comment:
just for eslint to be happy, it shouldn't change the code beahavior
```suggestion
    [dispatch, query],
```